README.md

```diff
@@ -80,38 +86,31 @@ E-Bikes is a sample application that demonstrates how to build applications with
-
-1. Click **Publish**, to publish the community. Click on the workspace icon in the top left corner, then click **View E-Bikes** to see the live community.
+1. Open the App Launcher, click **View all** then click on either of the **E-Bikes** cards to open the app or the Community.
 
-1. For experiencing the Salesforce app, open the App Launcher, and select the **E-Bikes** app.
+## Installing E-Bikes using a Developer Edition Org or a Trailhead Playground
 
-## Installing E-Bikes using a Developer Edition Org
+Follow this set of instructions if you want to deploy the app to a more permanent environment than a Scratch org.
+This includes non source-tracked orgs such as a [free Developer Edition Orgs](https://developer.salesforce.com/signup) or [Trailhead Playgrounds]().
 
-These steps assume you have followed the instructions above to install the application into a scratch org first, and now want to deploy it to a more permanent environment, including for completion of the [Lightning Web Components Basics Trailhead module](https://trailhead.salesforce.com/content/learn/modules/lightning-web-components-basics).
+Make sure to start from a brand-new environment to avoid conflicts with previous work you may have done.
 
-1. It's recommended to sign up for a [new Developer Edition org](https://developer.salesforce.com/signup), to avoid conflicts with work you may have done in any other orgs. If you created a new Developer Edition org to serve as a DevHub, and haven't done other work in the org, you can use that org.
+1. Log in to your org
 
-1. Once you've logged in to the org, in **Setup**, under **My Domain**, [register a My Domain](https://help.salesforce.com/articleView?id=domain_name_setup.htm&type=5).
+1. If you are setting up a Developer Edition: go to **Setup**, under **My Domain**, [register a My Domain](https://help.salesforce.com/articleView?id=domain_name_setup.htm&type=5).
 
 1. In **Setup**, under **Communities Settings**, click the **Enable communities** checkbox, and then select and register a subdomain for your community.
 
 1. In **Setup**, under **Communities Settings** and **Community Management Settings**, click the **Enable ExperienceBundle Metadata API** checkbox, and click **Save**.
 
 1. In **Setup**, under **Object Manager**, delete the custom **Product** picklist field from the **Case** object.
 
-1. At the command line, authenticate to your Developer Edition, and provide it with an alias (**ebikesDE** in the command below):
+1. Open a terminal, authenticate to your org, and provide it with an alias (**ebikesDE** in the command below):
 
     ```
     sfdx force:auth:web:login -a ebikesDE
     ```
 
-1. Check out a new branch of the code, to make changes that will allow deployment to a Developer Edition org:
-
-    ```
-    git checkout -b devOrg
-    ```
-
 1. In VS Code, use the Ctrl/Cmd-P shortcut for Quick Open. Type **E_Bikes.site** and click on the **E_Bikes.site-meta.xml** file to open it.
```

Comment:
I'm wondering if we shouldn't be more clear. Cmd+P for macOS, Ctrl+P for Windows.

Reply:
Agreed, fixed.